go-controller/pkg/util/net_linux.go

```diff
@@ -170,27 +171,36 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string, port int32) {
+func DeleteConntrack(ip string, port int32, protocol kapi.Protocol) {
 	ipAddress := net.ParseIP(ip)
 	if ipAddress == nil {
 		klog.Errorf("Value \"%s\" passed to DeleteConntrack is not an IP address", ipAddress)
```

Comment:
@JacobTanenbaum could you change the "\"%s\" to a %q while you're here?

Reply:
done